docker-compose/ci-cadence-dependencies.yml

```diff
@@ -23,24 +23,25 @@ services:
       - "9042:9042"
     networks:
       - testing-network
-  zookeeper:
```

Comment:
this is no longer available, see : https://github.com/uber/cadence/pull/5975

Reply:
cadence test is still failing, fixing the test env now

Reply:
Give up for now, still doesn't work: https://github.com/indeedeng/iwf/issues/381 for some reasons